### PR TITLE
Translate Ruby 2.0.0-p643 Released (ko)

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

**Comment by marocchino** [comment]
@namsk review please.

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+layout: news_post
+title: "루비 2.0.0-p643 릴리즈"
```
Comment (yous) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: '릴리스' is correct for 'release'. http://krdic.naver.com/detail.nhn?docid=12325000#FGN12434

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+date: 2015-02-25 12:00:00 +0000
+lang: ko
+---
+
+루비 2.0.0-p643이 릴리즈 되었습니다.
+
+이는 루비 2.0.0의 마지막 통상 릴리즈 입니다.
+앞으로 루비 2.0.0은 보안 유지보수 단계에 들어가, 중대한 회귀나 보안 문제를
+발견한 경우를 제외하고 릴리즈는 없습니다.
+이 단계는 1년동안 유지될 계획입니다.
```
Comment (yous) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: '1년 동안'

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+그러므로, 루비 2.0.0의 유지보수는 2016년 2월 24일에 종료됩니다.
+2.1이나 2.2버전같은 새로운 버전으로 이관을 검토하시기를 권장합니다.
```
Comment (yous) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: '버전 같은'

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+이 릴리즈는 많은 버그 수정이 포함됩니다.
+자세한 내용은 [티켓](https://bugs.ruby-lang.org/projects/ruby-200/issues?set_filter=1&amp;status_id=5)
+과 [변경로그](http://svn.ruby-lang.org/repos/ruby/tags/v2_0_0_643/ChangeLog)를
```
Comment (yous) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: '과' should be right after '티켓'.

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md (hunk 1 above)
Comment (marocchino) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: I am okey with '릴리스', but we have mush more '릴리즈' in this repo.
Maybe we should correct that later.

Comment (yous) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: Yes, I saw huge amount of '릴리즈'. We should replace all of them at once.

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+title: "루비 2.0.0-p643 릴리즈"
+author: "usa"
+translator: "marocchino"
+date: 2015-02-25 12:00:00 +0000
+lang: ko
+---
+
+루비 2.0.0-p643이 릴리즈 되었습니다.
+
+이는 루비 2.0.0의 마지막 통상 릴리즈 입니다.
```
Comment (yous) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: '릴리즈입니다'

File: ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md (hunk 1 above)
Comment (marocchino) [inline ko/news/_posts/2015-02-25-ruby-2-0-0-p643-is-released.md]: I made another PR #988  for this.

**Pull request merged**